data/json/items/gun/20x66mm.json

```diff
@@ -46,6 +46,10 @@
     "aim_speed" : 4,
     "durability": 9,
     "burst": 4,
```

Comment:
The (open) question is are we going to support both formats? When there are only two modes this isn't arguably any clearer than just specifying `burst` and this PR exemplifies well a potential error - the `BURST` mode should be named `AUTO`